content/docs/studio/get-started.md

```diff
@@ -75,25 +74,18 @@ details]).
    assignments.
 
    [models dashboard]:
-     /doc/studio/user-guide/model-registry/view-and-compare-models#models-dashboard
+     /doc/studio/user-guide/model-registry/view-models#models-dashboard
 
 2. Click on the model name to
-   [open its details page](/doc/studio/user-guide/model-registry/view-and-compare-models#model-details-page).
-
-3. You can perform the following actions to manage the lifecycle of models:
-   - [Add models to the registry](/doc/studio/user-guide/model-registry/add-a-model)
-   - [Register new versions of the models](/doc/studio/user-guide/model-registry/register-version)
-     (like `v0.0.1` and `v1.0.0`)
-   - [Assign stages to model versions](/doc/studio/user-guide/model-registry/assign-stage)
-     (e.g. `dev`, `testing`, `prod`)
-   - Use models:
-     [download them](/doc/studio/user-guide/model-registry/use-models) or
-     [set up CI/CD](/doc/studio/user-guide/model-registry/use-models) to publish
-     or deploy models
-
-Check out
-[`What is a model registry?`](/doc/studio/user-guide/model-registry/what-is-a-model-registry)
-to get started.
+   [open its details page](/doc/studio/user-guide/model-registry/view-models#model-details-page).
```

Comment:
I think it's best to make MR changes in a separate PR and limit this PR for exps changes only. wdyt?

Reply:
Sorry, I think this was an accident. Not sure where these changes came from. Let me figure it out and revert.